### PR TITLE
Created Site Performance Troubleshooting

### PR DESCRIPTION
**Pull request by TheGeekyUnicorn**
Added the troubleshooting guide an created a link for it on Troubleshooting Index page

### CODE REVIEW

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+versionFrom: 8.0.0
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]: This applies to Umbraco 7 as well, right?
If it does, we should just use versionFrom: 7.0.0 - what do you think?

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+# Site Performance checklist
+
+CPU issues can be many different things! Below I will give some advice on narrowing down the issue and trying to find out what is causing the problems.
+The very first thing to do, is try to find out when the trouble started. The friendly Umbraco SWAT will supply you with a graph of your CPU usage for the past 24 hours. Here you will likely notice a spot where it starts going up.
+
+## Resources that can help you:
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
## Resources that can help you
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+versionFrom: 8.0.0
+---
+
+# Site Performance checklist
+
+CPU issues can be many different things! Below I will give some advice on narrowing down the issue and trying to find out what is causing the problems.
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
CPU issues can be many different things. Below I will give some advice on narrowing down the issue and trying to find out what is causing the problems.
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+If you have Google Analytics (GA) set up for your site, then you should check things like visitor counts, and the specific pages visited during the CPU increase.
+Be aware that things like poorly implemented search functionality can be really bad for performance, so if you have a lot of during that time it could point to that.
+If you have a lot more visitors than normally, then maybe your site is back to normal now, but then a load test would probably be a good idea!
+
+### UmbracoTraceLogs
+
+If you know when your site started having issues, then you should compare that time to your Umbraco log. Often you will find errors that help you narrow it down, or other things that could be the cause.
+If for example the CPU performance issues started when you are building your search indexes then there is a good chance that the two are related.
+Be aware that some errors may appear due to maxing out your resources, they may not be related to the cause!
+Note: If you are unsure which log file corresponds to the time, you can get a list of the latest log files in Kudu by going to the logs folder `~/wwwroot/App_Data/logs and then typing in`:
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
Note: If you are unsure which log file corresponds to the time, you can get a list of the latest log files in Kudu by going to the logs folder `~/wwwroot/App_Data/logs` and then typing in:
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+versionFrom: 8.0.0
+---
+
+# Site Performance checklist
+
+CPU issues can be many different things! Below I will give some advice on narrowing down the issue and trying to find out what is causing the problems.
+The very first thing to do, is try to find out when the trouble started. The friendly Umbraco SWAT will supply you with a graph of your CPU usage for the past 24 hours. Here you will likely notice a spot where it starts going up.
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]: Perhaps we should add something about how to get in touch with support?
And I think we should call it "support" and not SWAT, as not every one might know that's the support team 😉

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+`ls -t | head -5`
+
+This will sort the files by last edit time, and only show you the top 5 results.
+
+## Common issues
+
+You can find a list of general performance anti-patterns in Umbraco here: https://our.umbraco.com/documentation/Reference/Common-Pitfalls/ 
+
+## In addition to that, here are some things we see often:
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
## In addition to that, here are some things we see often
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+Be aware that some errors may appear due to maxing out your resources, they may not be related to the cause!
+Note: If you are unsure which log file corresponds to the time, you can get a list of the latest log files in Kudu by going to the logs folder `~/wwwroot/App_Data/logs and then typing in`:
+
+`ls -t | head -5`
+
+This will sort the files by last edit time, and only show you the top 5 results.
+
+## Common issues
+
+You can find a list of general performance anti-patterns in Umbraco here: https://our.umbraco.com/documentation/Reference/Common-Pitfalls/
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+If you know when your site started having issues, then you should compare that time to your Umbraco log. Often you will find errors that help you narrow it down, or other things that could be the cause.
+If for example the CPU performance issues started when you are building your search indexes then there is a good chance that the two are related.
+Be aware that some errors may appear due to maxing out your resources, they may not be related to the cause!
+Note: If you are unsure which log file corresponds to the time, you can get a list of the latest log files in Kudu by going to the logs folder `~/wwwroot/App_Data/logs and then typing in`:
+
+`ls -t | head -5`
+
+This will sort the files by last edit time, and only show you the top 5 results.
+
+## Common issues
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
## [Common issues](https://our.umbraco.com/documentation/Reference/Common-Pitfalls/)
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+### Accessing the database
+
+Doing operations against your database can become a performance issue if you do it in the wrong way. If you for example get some data from the database in your views, then you may do a database operation for each visitor to that page. Can quickly become a problem.
+Please be aware that using any Umbraco Service will also include the database, and often the info you need can be retrieved from the cache instead!
+
+### Scheduled jobs
+
+Scheduled jobs can be very nice, but you have to think about how often they should run and how taxing they are on your sites performance. If for example you want to rebuild your cache every 5 minutes but it takes 6 minutes to do so you will run into a lot of trouble!
+
+- Reference Umbraco 7: https://our.umbraco.com/documentation/Reference/Config/umbracoSettings/#scheduledtasks
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
- [Reference Umbraco 7](https://our.umbraco.com/documentation/Reference/Config/umbracoSettings/#scheduledtasks)
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+- Reference Umbraco 8: https://our.umbraco.com/documentation/Reference/Scheduling/
```
Comment (sofietoft) [inline Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]:
```suggestion
- [Reference Umbraco 8](https://our.umbraco.com/documentation/Reference/Scheduling/) 
```

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md (hunk 1 above)
Comment (TheGeekyUnicorn) [reply Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]: Will use versionFrom 7.0.0 instead :)

File: Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md (hunk 5 above)
Comment (TheGeekyUnicorn) [reply Umbraco-Cloud/Troubleshooting/troubleshooting-siteperformance.md]: SWAT has been changed to Support and information on how to get in touch with us as well.

**Pull request merged**